doc/stdlib/dates.rst

```diff
@@ -136,6 +136,11 @@ alternatively, you could call ``using Dates`` to bring all exported functions in
    Returns a DateTime corresponding to the user's system
    time as UTC/GMT.
 
+.. function:: eps(::DateTime) -> Millisecond
+              eps(::Date) -> Day
+
+   Returns ``Millisecond(1)`` for ``DateTime`` values and ``Day(1)`` for ``Date`` values.
+
```

Comment:
You're saying it look like `Base.eps(::DateTime...` instead of just `eps` because as it, it seems to imply that you need to do `Dates.eps`? I guess that makes sense, but I also don't see any harm in it. Fully qualifying a name to make sure you get the method you want seems like good, conservative behavior. Or is there some other doc-related problem here?